pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -133,8 +134,12 @@ def get_coupled_variables(self, variables):
 
         elif SEI_option == "electron-migration limited":
             # Scott Marquis thesis (eq. 5.94)
-            eta_inner = delta_phi - phase_param.U_inner
-            j_sei = (eta_inner < 0) * phase_param.kappa_inner * eta_inner / L_sei_inner
+            j_sei = 0
+            if self.double_layer_sei:
+                eta_inner = delta_phi - phase_param.U_inner
+                j_sei = (
+                    (eta_inner < 0) * phase_param.kappa_inner * eta_inner / L_sei_inner
+                )
```

Comment:
electron-migration limited should still work with the single sei layer, we would just assume that the single layer is the inner layer

Reply:
> we would just assume that the single layer is the **inner layer**

All right, but in the single-layer case we usually assume that the layer to be used is the outer layer right? Should we make an exception for this case and continue with the previous assumption in the remaining code?

Reply:
If we just make it a separate file it will be easier as there won't be a need for an exception